cross/tcl/Makefile

```diff
@@ -1,36 +1,92 @@
-PKG_NAME = tcl
-PKG_VERS = 8.6.6
-PKG_EXT = tar.gz
-PKG_DIST_NAME = $(PKG_NAME)$(PKG_VERS)-src.$(PKG_EXT)
-PKG_DIST_SITE = http://prdownloads.sourceforge.net/tcl/
-PKG_DIR = $(PKG_NAME)$(PKG_VERS)/unix
+PKG_NAME	= tcl
+PKG_VERS	= 8.6.6
+PKG_EXT		= tar.gz
+PKG_DIST_NAME	= $(PKG_NAME)$(PKG_VERS)-src.$(PKG_EXT)
+PKG_DIST_SITE	= http://prdownloads.sourceforge.net/tcl/
+PKG_DIR		= $(PKG_NAME)$(PKG_VERS)/unix
 
-DEPENDS =
-BUILD_DEPENDS = cross/zlib
+DEPENDS		=
+BUILD_DEPENDS	= cross/zlib
```

Comment:
Generally speaking, dependencies are actual runtime dependencies, and should be added to `DEPENDS`. Is there any particular reason/documentation on why zlib is a build-time dependency only for TCL?